### PR TITLE
Make Ripley great again (Part 1)

### PR DESCRIPTION
**Pull request by InsaneHyena**
:cl:
balance: Buffs Ripley by essentially merging it with the Firefighter.
/:cl:

As it is right now, Ripley is pretty much garbage. It's slow, it's bad at fighting lavaland fauna, and it's just not worth the effort to make it compared to walking on foot. I think that it's a travesty, and a mining mech should be great at mining - so this is one of the few changes I want to make to it to make Ripley great again. 
Keep in mind that Firefighter mech still exists (for wreckage purposes) - it just can't be made at exosuit fabricator anymore, since Ripley is now the same mech, but slightly better (has a little bit more tools available and is cheaper to make).

I think I didn't fuck up anything this time.

### CODE REVIEW

**Comment by PopNotes** [comment]
yeah I think this is fair, there is literally no reason to ever build the ripley as long as the firefighter exists

**Comment by Repukan** [comment]
It's intended to suck at fighting fauna because fighting fauna it's not mining, it's basically hunting/poaching.

**Comment by KingPhilipIII** [comment]
@Repukan Problem with that is no matter where you go you encounter fauna that will proceed to wreck your shit.Maybe it shouldn't be buffed to the level of being used to hunt for fauna, but you should be able to hold your own.

**Comment by InsaneHyena** [comment]
But fighting fauna is an essential part of mining.

**Comment by Repukan** [comment]
No it isn't.

You can mine until round end without having to fight fauna.

**Comment by InsaneHyena** [comment]
Maybe, if you go out of your way to run like a coward, but it's not fun. "It's INTENDED to suck" is a pretty dubious statement, and even if it's true, than the intention behind it is bad. Combat mechs excel at killing people. Odysseus excels at healing people. Ripley should be great at mining - and killing fauna (though not necessarily megafauna) is a part and parcel of mining.

**Comment by Repukan** [comment]
Ripley excels at mining. It can mine faster and more than miners usually can, even with plasma cutter, and make wider paths to get more ore and glass.

Don't balance based on fun because fun is subjective. The person that enjoyed old mining and had fun in that slow environment won't agree with the same person that finds lavaland fun.

Ripley was suppose to be an upgrade during the time of asteroid mining. It still excels in mining, but it not being able to deal with fauna easily it's intended. If it has no drawbacks, then it just becomes a second health bar, with no Atmos resistance and now lava walking.

**Comment by kevinz000** [comment]
Great so now people can rampage even harder with a drill yay

**Comment by Ergovisavi** [comment]
Fact: The ripley's design is fundamentally flawed due to the way our mineral economy is set up, and any attempts to improve it for use on lava land is basically a huge waste of time.

The ripley is a mech designed to strip mine. Except strip mining is completely, utterly fucking useless, due in part to low mineral requirements on everything that science produces, ORM upgrades (Fuck whoever put these in), material tech levels, etc. The mineral requirements on objects were never properly tuned and as a result, the Ripley has no actual place in the game.

The ripley was ALSO useless in the original implementation of the asteroid because of the existence of the field to the southwest, which had a higher surface area and had high yield tiles, meaning drilling through large amounts of ores was also stupid.

We also, at this point, have the advanced plasma cutter, which mines faster than the ripley ever could, allows you to move on foot at full speed, fits in your backpack, acts as a ranged weapon, etc.

The ripley simply has no hole to fill. Attempting to artificially produce a purpose without facilitating an actual gameplay role is a fool's journey.

**Comment by DoctoreMad** [comment]
>As a side note "Make ripley great again" implies it was ever good. It wasn't.

~~Neither was America yet-~~

**Comment by Repukan** [comment]
Ex-fucking-cuse me

**Comment by kilkun** [comment]
@kevinz000 ripleys are like super fragile just nail it with an emp until the unarmored idiot flies out

**Comment by InsaneHyena** [comment]
You shut your whore mouth, Mad, roboticist-miner on Bay was my first experience with ss13, and Ripley will forever be my waifu

File: code/game/mecha/working/ripley.dm [hunk 1]
```diff
@@ -5,11 +5,12 @@
 	step_in = 4 //Move speed, lower is faster.
 	var/fast_pressure_step_in = 2 //step_in while in normal pressure conditions
 	var/slow_pressure_step_in = 4 //step_in while in better pressure conditions
-	max_temperature = 20000
-	max_integrity = 200
+	max_temperature = 65000
+	max_integrity = 250
+	resistance_flags = LAVA_PROOF | FIRE_PROOF | ACID_PROOF
 	lights_power = 7
+	armor = list(melee = 40, bullet = 30, laser = 30, energy = 30, bomb = 60, bio = 0, rad = 0, fire = 100, acid = 100)
```
Comment (Cyberboss) [inline code/game/mecha/working/ripley.dm]: quote these associations

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by optimumtact** [comment]
You don't need to assign Kor for every balance change cyberboss, especially one that isn't attracting any arguments against it

File: code/game/mecha/working/ripley.dm (hunk 1 above)
Comment (InsaneHyena) [inline code/game/mecha/working/ripley.dm]: I don't know what that means

**Comment by KorPhaeron** [comment]
Ergo makes such smart posts

**Comment by KorPhaeron** [comment]
Is giving miners a second healthbar that they can maintain with a welder and the ability to straight up ignore obstacles going to actually improve their experiences or leave them bored?

File: code/game/mecha/working/ripley.dm (hunk 1 above)
Comment (Cyberboss) [inline code/game/mecha/working/ripley.dm]: melee -> "melee" etc

**Comment by Shadowlight213** [comment]
Just make the ripley into an engineering mech instead of a mining mech. Rip out the mining equipment for it, and make the engineering ones cheaper.
Merging it with firefighter works well. Make the mech rcd place several floors at once, maybe walls too.
Make the extinguisher spray a huge area, with a giant, maybe infinite, internal tank.
Add thrusters to it so it can alightly maneuver in space. Stuff like that.

**Comment by Kraseo** [comment]
the mech extinguishers DO spray a bigger area than normal

**Comment by KorPhaeron** [comment]
@insanehyena

What other changes do you plan?

I share the concern that the ripley has no niche to fill so the only possible way to make it worth using is to just make it straight up better than miners

An upgrade for the sake of it existing

**Comment by KorPhaeron** [comment]
If its going to be

-Faster at mining

-Better at fighting (I assume this is part 2) and immune to goliath tentacles/watcher beams

-Immune to storms

-Immune to lava

What challenge or gameplay is even left? You get in your big box and thats it, things cant hurt you anymore

**Comment by kilkun** [comment]
I mean yeah thats kinda the idea behind a tank

**Review by Cyberboss — APPROVED** [review]

**Comment by KorPhaeron** [comment]
Yeah thats the idea of a tank if you are designing a real vehicle to make life easier and safer and not trying to make a videogame

If people dislike the gameplay they can avoid the job, there shouldnt be an option to collect all the rewards without engaging

**Comment by InsaneHyena** [comment]
The second part is giving Ripley a kinetic accelerator. 

I disagree with your opinion on how these changes (especially part 1 exclusively) will make Ripley overpowered, since Firefighter already exists, and nobody ever bothers to use it anyway. It doesn't seem like anybody shares your concerns either.

**Comment by Repukan** [comment]
>Giving the Ripley a KA

Wipe this meme from the face of the Earth.

**Comment by KorPhaeron** [comment]
@insanehyena

People who share my concerns and play a large amount of mining are posting in this thread though

**Comment by kilkun** [comment]
except thats exactly what the ripley was with the asteroid. this is simply giving the ripley the love it was lacking when everything else mining got an overhaul.

**Comment by Repukan** [comment]
Why don't you just drop pretenses and give miners ash proofing, vacuum immunity and a second health bar

**Comment by KorPhaeron** [comment]
If the ripley was a magic safety box on the asteroid it shouldnt need buffs because you're fighting essentially the same monsters

But the ripley was garbage on the asteroid because you'd finish mining the high yield zone and be done mining for the round by the time robotics could print one out

**Comment by KorPhaeron** [comment]
Interesting how one by one people in favour of this change are admitting they hate the job and dont care how it impacts it

**Comment by kilkun** [comment]
i hate every job that isn't assistant or captain, that's hardly a fair metric to make

**Comment by Cyberboss** [comment]
I'm gonna close this soon @KorPhaeron if you don't

**Comment by PopNotes** [comment]
reopen this and merge it

**Comment by optimumtact** [comment]
go back to hippie

**Comment by kilkun** [comment]
reopen this and merge it

**Pull request closed without merging**